docs/UserGuide.md

```diff
@@ -310,12 +310,9 @@ This feature involves the command: `updatephoto`, which updates the photo of an
 
 Format: `updatephoto PERSON_INDEX [path/NEW_PHOTO_PATH]`
 
-You can get the path to the photo by using the following steps: 
-  * Right click on the photo that you want to use.
-  * Choose the option: Copy as path (on Windows) or Copy "name_of_file" as Pathname (on MacOS).
-  * After pasting the copied path in CampusConnect, remember to remove the double quotation marks!
-
-  ![photo_path](images/photopath.png)
+To obtain the `NEW_PHOTO_PATH` of your photo: 
+* If you are a Windows user, click [here](https://www.timeatlas.com/copy-file-path/#:~:text=Select%20your%20file%2C%20right%2Dclick,Ctrl%20%2B%20Shift%20%2B%20%2B%20C%20).
+* If you are a macOS user, click [here](https://technastic.com/copy-file-path-mac/).
```

Comment:
Given you have written that "To obtain the `NEW_PHOTO_PATH` of your photo", the "click here" somewhat sounds like you can actually get the path from there. Although I doubt anyone feels the same way in real world scenario, you can consider change it to "learn from here".

It is very minor so no stress 😄

Reply:
yes agreed! was not very sure how i can phrase it better, good point raised! fixed in 0f2693b7